litgpt/chat/base.py

```diff
@@ -196,6 +197,10 @@ def main(
         compile: Whether to use compilation to speed up token generation. Will increase startup time.
         multiline: Whether to support multiline input prompts.
     """
+    if not checkpoint_dir.is_dir():
+        checkpoint_dir = "checkpoints" / checkpoint_dir
+    pprint(locals())
```

Comment:
Is this intentional? It wouldn't make much sense for chat, right?

Reply:
> You could add these lines of code to a function and reuse it. It would also be easier to test it this way.

Sounds, good! I will refactor!

> Is this intentional? It wouldn't make much sense for chat, right?

You mean the `pprint(locals())`? Yes, I think it's always useful to know the settings you are running. It will help detecting wrongs paths, etc.